src/vl-mapactions-delete-action.js

```diff
@@ -3,6 +3,7 @@ import {VlBoxSelectAction} from './vl-mapactions-box-select-action';
 
 export class VlDeleteAction extends VlBoxSelectAction {
   constructor(layer, onDelete, options) {
```

Comment:
Als je option default een leeg object wil maken kan dat op deze manier;
`constructor(layer, onDelete, options = {}) {`

Reply:
Goede opmerking. 

Het nadeel van regelmatig te switchen tussen code bases die "nieuwe" (welja, nieuw is het ondertussen niet meer) ECMAScript versies niet ondersteunen en code bases zoals deze die dat wel doen is dat ge soms niet meer weet wat je waar mag gebruiken :)